src/Farmer/Builders/Builders.Dns.fs

```diff
@@ -175,11 +221,20 @@ type DnsPtrRecordBuilder() =
 
     /// Builds a record for an existing DNS zone.
     [<CustomOperation "link_to_unmanaged_dns_zone">]
-    member _.LinkToDnsZone(state:PtrRecordProperties, zone:ResourceId) = { state with Zone = Some (Unmanaged zone) }
+    member _.LinkToUnmanagedDnsZone(state:PtrRecordProperties, zone:ResourceId) = { state with Zone = Some (Unmanaged zone) }
+
+    /// Builds a record for an existing DNS zone that is managed by this Farmer deployment.
+    [<CustomOperation "link_to_dns_zone">]
+    member _.LinkToDnsZone(state:PtrRecordProperties, zone:ResourceId) = { state with Zone = Some (Managed zone) }
```

Comment:
This repeated pattern feels like a bit of boilerplate to me. Could we use the same trick we have done for e.g. `IDependable` etc. in order to get these three or four keywords + overloads in one place? It could even be inside this same file as all the DNS types are here anyway.

Reply:
That works for me :-)